charts/renku/values.yaml

```diff
@@ -632,11 +632,24 @@ notebooks:
     #   every: 60
     singleuser:
       image:
-        name: renku/singleuser
```

Comment:
Is this on purpose? just asking because it's a change that has nothing to do with the rest of this PR.

Reply:
yes I got tired of looking at an image ref that is >1 year old